src/components/DropdownMenu/styles.tsx

```diff
@@ -0,0 +1,104 @@
+import COLORS from '@/styles/colors';
+import styled from 'styled-components';
+
+export const MenuContainer = styled.div<{ $show: boolean }>`
+  position: absolute;
+  top: 100%;
+  left: 0;
+  transform: translateY(2px);
+
+  display: ${({ $show }) => ($show ? 'flex' : 'none')};
+  flex-direction: column;
+  width: max-content;
+  min-width: 100%;
+  max-height: 12.5rem;
+  overflow-y: auto;
+  background: white;
+  box-shadow: 0 2px 0.25em 0.1em rgba(0, 0, 0, 0.25);
+  border: 1px solid rgba(0, 0, 0, 0.25);
+  padding: 0.2rem 0.3rem;
+  border-radius: 0.5rem;
+  z-index: 100;
+
+  &:empty {
+    &::after {
+      content: 'No matches found!';
```

Comment:
two things:
- nit: can we get rid of the exclamation point in the no matches found message
- when i type something in and then delete it completely, it seems to go back to this "no matches found" state. is this the expected behavior? i think it might be better to display every option in the dropdown, since the search string is empty

Reply:
yes 🫡 thanks for the bug catch

Reply:
as for the second one, I don't think I'm experiencing this? After deleting the query, it displays all the options for me.

Reply:
Ah, I think I know what you're talking about. If you type in a query to search, then select an option, your query will be deleted but the filter will remain. This is intended. Two reasons:
1. The user can keep typing to search without having to manually delete their existing query
2. The user may want to select more options in their existing query

Reply:
hmm i think it might be different? i.e. i typed a bunch of gibberish without selecting any options, the menu said "no matches found", and then i backspaced completely and it still said "no matches found"
<img width="655" alt="Screenshot 2023-11-19 at 3 57 25 PM" src="https://github.com/calblueprint/immigration-justice-project/assets/47801900/8390ff9c-041a-4660-b595-9ba17cd2739c">

Reply:
I found a way to reproduce the screenshot you sent by pasting some random code with trailing new lines. I implemented a fix to trim white spaces, which seems to fix it. Let me know if the issue persists!

Reply:
ahdjhjdewd it seems to still be there :( let's also test for this at worksesh tmr!!